pulp-glue/pulp_glue/core/context.py

```diff
@@ -125,6 +125,18 @@ def HREF(self) -> str:  # type:ignore
             return "auth_group_href"
         return "group_href"
 
+    def add_user(self, user: "PulpUserContext") -> None:
+        pass
+
+    def remove_user(self, user: "PulpUserContext") -> None:
+        user_pk = user.pulp_href.split("/")[-2]
+        group_user_ctx = PulpGroupUserContext(self.pulp_ctx, self)
+        # This is a bad workaround, because the api is terrible.
```

Comment:
Some day : https://github.com/pulp/pulpcore/issues/5110

Reply:
At least we have a common place for weird workarounds these days.